docs/section19/19-17.md

```diff
@@ -0,0 +1,39 @@
+# Section 19 - Rule 19-17           
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** True    
+**Rule ID:** 19-17           
+**Rule Description:** For baseline system types 9 & 10, the system design supply airflow rates shall be based on the temperature difference between a supply air temperature set point of 105°F and the design space-heating temperature set point, the minimum outdoor airflow rate, or the airflow rate required to comply with applicable codes or accreditation standards, whichever is greater.  
+
+**Rule Assertion:** Options are Pass/UNDETERMINED/NOT_APPLICABLE    
+**Appendix G Section:** G3.1.2.8.2           
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:**  
+1. Applies to baseline HVAC system types 9 & 10.     
+   
+   
+**Function Calls:**  
+1. get_baseline_system_types  
+
+## Rule Logic:   
+**Applicability Check 1 - Check if there are any Applicable Baseline System Types in the Project at All**   
+- Create dictionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMI)`  
+- Check if B-RMR is modeled with at least one air-side type 9 or 10, if yes then carry on, if no then NOT_APPLICABLE for the project: `for sys_type in ["SYS-9","SYS-9b","SYS-10"]):`
+    - Check if values are not none for system type key in the dictionary, if it does not equal false then the values are not none and there are applicable system types in the project: `if bool(baseline_system_types_dict.get(sys_type)) != False: is_there_applicable_sys_type = TRUE`
+- Check if is_there_applicable_sys_type == True which means there are applicable system types in the project, if yes then carry on, if no then NOT_APPLICABLE for the project: `if  is_there_applicable_sys_type == True:`    
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `If any(hvac.id in baseline_hvac_system_dict[sys_type] for sys_type in ["SYS-9","SYS-9b","SYS-10"]): `    
+            - Get the supply temperature setpoint: `supply_air_temp_setpoint = hvac.fan_system.supply_air_setpoint_temperature` 
+
+            - **Rule Assertion:** 
+            - Case 1:If the supply air temperatures setpoint equals 105F : `if supply_air_temp_setpoint == 105: outcome = "PASS"`  
```

Comment:
I think we need to also check whether fan flow is autosized (need to add schema element for this). Also, should check that FanSystem.minimum_airflow>FanSystem.Minimum_Outdoor_Airflow (FAIL when this doesn't hold, unless AllowMoreLenient flag is set to TRUE). I think we also should add a note to the schema that the minimum_outdoor_airflow should be reported even when the fan flow is autosized, so this value will be based on simulation results for the sizing runs. (Same goes for autosized heating and cooling capacities.)

Reply:
All done except did not implement the allowmorelenient flag per discussion on the call.